spack.yaml

```
@@ -51,7 +40,5 @@ spack:
       tcl:
         include:
           - access-test
-          - oasis3-mct
         projections:
-          access-test: '{name}/2025.01.1'
-          oasis3-mct: '{name}/2023.11.09-{hash:7}'
+          access-test: '{name}/2025.03.1'
```

Should this be:
```suggestion
          access-test-model: '{name}/2025.03.1'
```
Given the spec? 

And is `access-test-model` meant to be a model component or a model?

If it's a component, it should probably go under `spack.packages` rather than `spack.specs`, since we use the `spack.specs[0]` as the deployment repo version - in which case the spec should be `access-test` instead.

---

Fixed in [50cf23e](https://github.com/ACCESS-NRI/ACCESS-TEST/pull/30/commits/50cf23e31ae6b3d9e273c934e7a1d7e740bb4c7c)